### ISSUE TITLE
Preflight reports missing dependencies in SCSS files on Windows

### ISSUE THREAD

**Issue by ProchaLu** [upleveled/preflight#443]
The "No dependency problems" check fails on Windows with `create-react-app` projects using SCSS:

```
⚠ No dependency problems
  ✖ No unused dependencies
    › Missing dependencies found:
    * C: .\src\styles.module.scss
    Add these missing dependencies by running the following command for each dependency:
    ‎  $ pnpm add <dependency name here>
```

No issue on macOS or Linux. Maybe related to the backslashes are causing this issue with the path?

**Comment by karlhorky** [comment]
This check is performed by [`depcheck`](https://github.com/depcheck/depcheck), so that repo is maybe a good place to follow up with that.

https://github.com/upleveled/preflight/blob/4057ea99986fbdaaa6ed4d722349828061cdb3c0/src/checks/noDependencyProblems/noUnusedDependencies.ts#L59-L62

**Comment by karlhorky** [comment]
Fixed in Depcheck `v1.4.6` (released in Preflight `v2.1.17`)

- https://github.com/depcheck/depcheck/releases/tag/v1.4.6
- https://github.com/upleveled/preflight/releases/tag/v2.1.17
